docs/verifier/query-builder.md

```diff
@@ -218,6 +218,10 @@ On-chain verification enables smart contracts to execute logic based on query re
 3. **Smart Contract Address:** Input the address of the contract responsible for proof validation and logic execution on the selected network.
 4. **Click the Set Request Button:** This action sends a `Set ZKP Request` transaction.
 
+:::caution
+Any verifier address can be used to set requests. However, only three specific validator unified addresses are hardcoded into the Query Builder. This means the verifier must have these addresses whitelisted; otherwise, the setRequest operation will fail. For our UniversalVerifier, this is not an issue since these validators are already included, but it's important to keep in mind for custom verifiers (embedded or standalone).
```

Comment:
We need a link to the Validators. May be to the smart-contract page

Reply:
Added